common/src/main/scala/lucuma/react/common/scalaComponents.scala

```diff
@@ -89,11 +89,6 @@ class ReactPropsWithChildren[Props, S, B](
   val component: Scala.Component[Props, S, B, CtorType.PropsAndChildren]
 ) extends ReactComponentProps[Props, S, B, CtorType.PropsAndChildren]
 
-object ReactPropsWithChildren:
```

Comment:
The extension methods in this file were added in #739. The compiler is warning that they will never be invoked because there is already an `apply` method with compatible parameters. Indeed, the extension just seems to be invoking the existing `apply` method with exactly the same parameters it was passed, so I just removed them.